pyproject.toml

```diff
@@ -55,28 +58,24 @@ exclude = '''
 
 [tool.ruff]
 line-length = 88
+
+[tool.ruff.lint]
 select = ["E", "F", "I", "Q", "W"]
+
 # line too long; Black will handle these
 ignore = ["E501"]
 
-[tool.ruff.mccabe]
 # Unlike Flake8, default to a complexity level of 10.
-max-complexity = 10
+mccabe = {max-complexity = 10}
```

Comment:
I'd rather stick to the ini-style `[tool.ruff.mccabe]` (etc) headings - much more readable